packages/editor/src/client/routes/comments/commentList.tsx

```diff
@@ -554,7 +563,7 @@ function CommentList() {
                               displayOnly
                               // TODO: remove this
                               onChange={console.log}
-                              value={text}
+                              value={text || []}
```

Comment:
This will not work as an empty array is invalid for Slate. See https://github.com/wepublish/wepublish/blob/master/packages/editor/src/client/routes/comments/commentEditView.tsx#L31